.pipelines/pr-e2e.yaml

```diff
@@ -45,16 +45,17 @@ jobs:
     displayName: 'Set up the Go workspace'
   - script: make bootstrap
     displayName: Install dependencies
-  - script: make validate-dependencies
-    displayName: Check if imports, Gopkg.toml, and Gopkg.lock are in sync
   - script: make ensure-generated
     displayName: Check if generated code is up to date
   - script: make generate test-style
     displayName: Run linting rules
   - script: make build-cross
     displayName: Build cross-architectural binaries
+  - script: make validate-dependencies
+    displayName: Check if go mod is up to date
   - script: |
         export CODECOV_TOKEN=$(CODECOV_TOKEN)
+        grep github.com/Masterminds/godir ./go.mod || echo "github.com/Masterminds/godir not in go.mod"
```

Comment:
Why are we checking for Masterminds/godir specifically here?

Reply:
I'm debugging :)